### PR TITLE
Alert updates

### PR DESCRIPTION
**Pull request by macandcheese**
Resolves https://github.com/Esri/calcite-components/issues/570 cc @jwasilgeo - can you give this a look over? It should work as desired, while still maintaining a queue if multiple are added to DOM with active present.

Resolves https://github.com/Esri/calcite-components/issues/456

**BREAKING**
- 🚨 - removes `open` and `close` methods in favor of `active` prop
- 🚨 - removes public `currentAlert`, `alertQueue`, and `alertQueueLength` props (queue and opened / closed element available as event details)

**ADDED**
- 🆕 - adds `intlClose` prop for translating button close label

**CHANGES**
- ♻️ - No longer assigns GUID or relies on internal alertId string for queue and active state management
- ♻️ - Updates tests, dev demos, Storybook

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [Unreleased]
```
Comment (paulcpederson) [inline CHANGELOG.md:8]: Gotta add the link below if you want this to be a link like:

[Unreleased]: https://github.com/Esri/calcite-components/compare/v1.0.0-beta.32...master "Unreleased"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ Please refer to the [Stencil testing documentation](https://stenciljs.com/docs/t
```
Comment (paulcpederson) [inline CONTRIBUTING.md:50]: I think you inadvertently reverted this file to its previous state

**Comment by paulcpederson** [comment]
@macandcheese looks like you are changing a lot of stuff from the `master` branch back to what it was like a couple days ago. You may want to rebase off of master or make a new branch from master and cherry pick your commits over because I think something got screwed up in your merge.

**Comment by paulcpederson** [comment]
It's your first "update changelog" commit I think : https://github.com/Esri/calcite-components/pull/739/commits/a12848fd78512e4a4c813e4b8119ed6bcf6e81f2

**Comment by macandcheese** [comment]
Weird, bad merge. Will reopen in a few.

**Pull request closed without merging**